<ui version="4.0" >
 <class>TopolDialog</class>
 <widget class="QDialog" name="TopolDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>340</y>
     <width>387</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QPushButton" name="checkButton" >
      <property name="text" >
       <string>Check Topology</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>200</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Check for:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>160</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QCheckBox" name="intersectBox" >
       <property name="text" >
        <string>Intersections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="overlapBox" >
       <property name="text" >
        <string>Overlaps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="multipartBox" >
       <property name="text" >
        <string>Multiparts</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="resultWindow" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TopolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TopolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
